free-3.md

```diff
@@ -10,41 +10,72 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A lot of people decorate their place with lights including lamps and string
+lights. Those lights are already manufactured and come in only certain
+styles, available at places like Ikea. I wish to change their experience by 
+making it easy for them to design their own lights so that they can submit 
+their own design to the manufacturers and get the customized lights.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Its domain is very specific to users who want to create their own design of 
+lights with ease. They would be provided with a lot of functions that let 
+them customize their lights, such as the number of bulbs, bulbs wrapped in 
+different shapes, mixing different shaped bulbs, length of the string, the 
+voltage, etc. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I love having lights, especially string lights, on the walls at my place. 
+When decorating my place, I sometimes wished for different designs of 
+the lights, such as having star-shaped covers for the bulbs and other shapes 
+alternate on the same string, as after some time I found the already-manufactured ones 
+tedious.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Designing and customizing lights
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+It can be run on the terminal. The user would be introduced with a list of 
+available functions and instructions first. 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The user can define a lights as a variable and call different functions on the
+variable for different settings such as shapes, brightness, length, number of bulbs, 
+etc. I imagine using Python as the domain language.  
```

Comment:
I think what you are describing fits slightly better under syntax versus semantics.